lib/common/utils/mixin/callback_after_build.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+
+/// A mixin that can be used with a [StatefulWidget] to register a callback.
+/// function [callbackAfterBuild] to be called after the widget is built.
+/// To use this mixin, create a class
+///  that extends [StatefulWidget] and mixes in.
+/// this class. Then, implement the [callbackAfterBuild]
```

Comment:
[nits]
若干、文章がおかしい気がするのですが、↓のようにするといいかもしれません。

```suggestion
///  that extends [StatefulWidget] and mixes in
/// that class. Then, implement the [callbackAfterBuild]
```